### PR TITLE
[jdbc-to-elasticsearch] Migrate akka dependencies to pekko

### PR DESCRIPTION
**Pull request by laglangyue**
this pr inclueds:
Using pekko instead of akka in document and code dependencies
Test:
test local by  `docker 24.0.7 on Mint Linux 21`

### CODE REVIEW

**Comment by laglangyue** [comment]
[info]  * org.apache.pekko:pekko-stream_2.13:1.0.2 is selected over {1.0.1, 1.0.1}
[info]      +- org.apache.pekko:step_001_complete_2.13:1.0.1      (depends on 1.0.2)
[info]      +- org.apache.pekko:pekko-connectors-slick_2.13:1.0.1+12-db689989-SNAPSHOT (depends on 1.0.2)
[info]      +- org.apache.pekko:pekko-connectors-elasticsearch_2.13:1.0.1+12-db689989-SNAPSHOT (depends on 1.0.2)


which version can be used for pekko-connectors of pekko-1.0.1

**Review by pjfanning — COMMENTED** [review]
Don't add license headers. I'll look at the license set up another time but we will probably not Apache license these sample files.

File: pekko-connectors-sample-jdbc-to-elasticsearch/project/Dependencies.scala [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
 
 object Dependencies {
   val scalaVer = "2.13.8"
-  // #deps
-  val AkkaVersion = "2.6.19"
-  val AlpakkaVersion = "4.0.0"
+
+  val pekkoVersion = "1.0.1"
+
+  val pekkoConnectorVersion = "1.0.1+12-db689989-SNAPSHOT"
```
Comment (pjfanning) [inline pekko-connectors-sample-jdbc-to-elasticsearch/project/Dependencies.scala]: Use the 1.0.2 release not a snaphot

File: pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java [hunk 2]
```diff
@@ -1,31 +1,39 @@
 /*
- * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
```
Comment (pjfanning) [inline pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java]: Revert all license changes. We need to keep the Lightbend headers.

Comment (laglangyue) [reply pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java]: done

File: pekko-connectors-sample-jdbc-to-elasticsearch/project/Dependencies.scala (hunk 1 above)
Comment (laglangyue) [reply pekko-connectors-sample-jdbc-to-elasticsearch/project/Dependencies.scala]: I did not find pekko connectors 1.0.2 in the Maven repository, so I used snapshot to test this sample locally and learn pekko streams. After the official release 1.0.2, I will change the versions of pekko and pekko connectors.

File: pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java (hunk 2 above)
Comment (laglangyue) [reply pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java]: I have removed apache license headers and keep the Lightbend headers. Friendly question: Why not add the Apache license header?

Comment (pjfanning) [reply pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java]: When a source file already has a license header, you normally just keep it. It's a big legal mine field and it's been one of the most complicated aspects of getting Pekko up and running. I am busy this weekend and don't have time to study the licenses. I will look at them next week.

**Comment by pjfanning** [comment]
I pushed a few changes but I'm going to ask that we stop working on this for a few days.

This repo seems to require https://scalacenter.github.io/course-management-tools/docs/install and I've never used that. I need to find time to work out how to use and see if I can get the docs built for the samples. There isn't much point in updating the samples if we can't get the docs published.

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**